src/main/java/CommandManager.java

```diff
@@ -0,0 +1,79 @@
+
+public class CommandManager {
+    //print Duke
+    //get user input
+    private String userInput;
+    private String key;
+
+    public static final String DIVIDER = "\t____________________________________________________________";
+
+    public static void sayHi() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.print(DIVIDER + '\n'
+                + " \t Hello! I'm Duke\n\t What can I do for you?\n" + DIVIDER + "\n\n");
+    }
+
+    public static void sayBye() {
+            System.out.println(DIVIDER + "\n\t Bye. Hope to see you again soon!\n" + DIVIDER);
+    }
+
+    public CommandManager() {
+        userInput = null;
+        key = null;
+    }
+
+    public String getUserInput() {
+
```

Comment:
Unnecessary empty lines here and in `setKey(String key)` method, can remove them. Same issue noted in other files as well